Readme.md

```diff
@@ -397,16 +427,18 @@ Creates the specified file for output. If the file already exists, it will be ov
 .closefile
 ```
 
-Closes the currently opened output file.
+Closes the currently opened output file. Local labels and `equ`s defined before this directive are discarded.
 
 ### Set the output position
 
 ```
-.org RamAddress
+.org  RamAddress
+org   RamAddress
 .orga FileAddress
+orga  FileAddress
 ```
 
-Sets the output pointer to the specified address. `.org` specifies a memory address, which is automatically converted to the file address for the current output file. `.orga` directly specifies the absolute file address.
+Sets the output pointer to the specified address. `.org`/`org` specifies a memory address, which is automatically converted to the file address for the current output file. `.orga`/`.orga` directly specifies the absolute file address. Local labels and `equ`s defined before this directive are discarded.
```

Comment:
`` `.orga`/`.orga\` `` has the same syntax repeated, I assume you mean `` `.orga`/`orga\` ``.

Reply:
Thanks; fixed this in a force push.